src/middlewares.js

```diff
@@ -110,7 +110,11 @@ export function handleParseHeaders(req, res, next) {
   req.config = new Config(info.appId, mount);
   req.info = info;
 
-  const ip = req.ip;
+  const ip = req.headers['x-forwarded-for'] ||
```

Comment:
Quick thing, it is possible that this value may be multiple ips. Especially if the user passed through one or more proxies before getting to a balancer, (quick breakdown [here](https://en.m.wikipedia.org/wiki/X-Forwarded-For)). 

Considering that _if_ you have more than 1 IP, and they are each separated by a `,`, you can just split the string apart by it's commas to and take the first 'origin' entry (or the only entry).
```js
req.headers['x-forwarded-for'].split(',')[0]
```